From ed45bb8661d3fbda3a2c59c342f7071c4f241926 Mon Sep 17 00:00:00 2001
From: Amit Shah <amit.shah@redhat.com>
Date: Wed, 16 Jun 2010 05:17:39 -0300
Subject: [PATCH 5/5] monitor: Remove drive_add command for Red Hat Enterprise Linux

RH-Author: Amit Shah <amit.shah@redhat.com>
Message-id: <6e693accd05dba7edc46f2bdd926fa68e86a1d7e.1276665204.git.amit.shah@redhat.com>
Patchwork-id: 9929
O-Subject: [RHEL6 PATCH v2 3/3] monitor: Remove drive_add command for Red Hat
	Enterprise Linux
Bugzilla: 602590
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>

drive_add is the old way of attaching storage to a PCI device.
A later series will add the 'blockdev_add' command that will be more
generic.

Signed-off-by: Amit Shah <amit.shah@redhat.com>
---
 qemu-monitor.hx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 qemu-monitor.hx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/qemu-monitor.hx b/qemu-monitor.hx
index a6bb1a0..063257b 100644
--- a/qemu-monitor.hx
+++ b/qemu-monitor.hx
@@ -831,7 +831,7 @@ STEXI
 Set maximum tolerated downtime (in seconds) for migration.
 ETEXI
 
-#if defined(TARGET_I386)
+#if defined(TARGET_I386) && 0 /* Disabled for Red Hat Enterprise Linux */
     {
         .name       = "drive_add",
         .args_type  = "pci_addr:s,opts:s",
-- 
1.7.0.3

